code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -482,7 +488,11 @@
 		else
 			. += "<span class='notice'>[copytext_char(temporary_flavor_text, 1, 37)]... <a href='?src=[REF(src)];temporary_flavor=1'>More...</a></span>"
 
-
+	//SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
+	var/datum/data/record/isinworld = find_record("name", perpname, GLOB.data_core.locked) //i dont know of a better way to do this-it doesnt work on off-station roles. please fix this someone better at coding.
+	if ((is_special_character(user)) && isinworld && (user.mind.has_antag_datum(/datum/antagonist/heretic/)) || (user.mind.has_antag_datum(/datum/antagonist/traitor/)) || (user.mind.has_antag_datum(/datum/antagonist/ninja/)) || (user.mind.has_antag_datum(/datum/antagonist/brother/)) || (user.mind.has_antag_datum(/datum/antagonist/changeling/)) || (user.mind.has_antag_datum(/datum/antagonist/malf_ai/)) || (user.mind.has_antag_datum(/datum/antagonist/revenant/)) || (user.mind.has_antag_datum(/datum/antagonist/wizard/))) //this is fucking terrible. i am so sorry. TODO: fix whatever the fuck this is
+		. += "<a href='?src=[REF(src)];exprecords=1'>\[View exploitable info\]</a>"
```

Comment:
```suggestion
	for(var/datum/antagonist/antag_datum in user?.mind?.antag_datums)
	    if(antag_datum.has_exploitables)
		    . += "<a href='?src=[REF(src)];exprecords=1'>\[View exploitable info\]</a>"
```
For "Antags I explicitly flag with a variable" (do this in a different, master file)

Reply:
complete!